themes/bootstrap3/js/embedded_record.js

```diff
@@ -84,17 +84,30 @@ VuFind.register('embedded', function embedded() {
     return true;
   }
 
-  function toggleDataView(_link, tabid) {
+  function toggleDataView(_link, _tabid) {
     var $link = $(_link);
+    var tabid = _tabid;
     var viewType = $link.attr('data-view');
     // If full, return true
     if (viewType === 'full') {
       return true;
     }
+    // Open tab
+    var openTab = false;
+    if ($link.data('open-tab')) {
+      openTab = $link.data('open-tab');
+    }
     var result = $link.closest('.result');
+    if (!$link.hasClass('title')) {
+      $link = result.find('.title');
+    }
     var mediaBody = result.find('.media-body');
     var shortNode = mediaBody.find('.short-view');
     var longNode = mediaBody.find('.long-view');
+    // Not title link
+    if (!$link.hasClass('title')) {
```

Comment:
This logic exists at lines 101-103 and lines 107-110; is the duplication intentional? It does not appear to be useful to repeat the behavior.